### PR TITLE
Docs: logging in the docker container

### LINKED ISSUE #23102: Docs: mention how logging in the docker container is configured
Information
**What Grafana version are you using?**
v6.5.2
**What datasource are you using?**
/
**What OS/platform are you running grafana on?**
kubernetes cluster
**What did you do?**
tried to access grafana log file `/var.log/grafana/grafana.log`
```bash
kubectl exec -it GRAFANA_POD -- ls -lsa /var/log/grafana
```
have default config from regarding logs (which is for log.mode= console file) as stated in https://grafana.com/docs/grafana/v6.5/installation/configuration/

**What happened instead?**
no grafana log file existent, which means that for default config file for docker image have log.mode= console
**Change request**
make it explicitly stated in the docs somewhere that default in docker image of grafana is only console for log.mode 

Thank you!

### PR DESCRIPTION
**Pull request by xlson**
**What this PR does / why we need it**:

Documents how logging is configured in the docker container and how to change it.

**Which issue(s) this PR fixes**:

Fixes #23102

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/installation/configure-docker.md [hunk 1]
```diff
@@ -53,6 +53,17 @@ GF_PATHS_LOGS         | /var/log/grafana
 GF_PATHS_PLUGINS      | /var/lib/grafana/plugins
 GF_PATHS_PROVISIONING | /etc/grafana/provisioning
 
+## Logging
+
+Logs in the docker container go to standard out by default, as is common in the docker world. Change this by setting a different [log mode]({{< relref "../administration/configuration.md#mode" >}}).
```
Comment (oddlittlebird) [inline docs/sources/installation/configure-docker.md:58]:
```suggestion
Logs in theDocker container go to standard out by default, as is common in the Docker world. Change this by setting a different [log mode]({{< relref "../administration/configuration.md#mode" >}}).
```

Comment (oddlittlebird) [reply docs/sources/installation/configure-docker.md:58]: What is "standard out by default"? Is this a developer term I am just unfamiliar with?

Comment (oddlittlebird) [reply docs/sources/installation/configure-docker.md:58]: Standard output? Does that mean console? If so, we should clarify.

**Review by oddlittlebird — APPROVED** [review]
Minor edits and one question.

File: docs/sources/installation/configure-docker.md (hunk 1 above)
Comment (xlson) [reply docs/sources/installation/configure-docker.md:58]: `Standard out` is a technical the term for the output stream that goes to the parent process (allthough, the correct term is probably standard output). If you start grafana from the terminal the terminal will be the parent process and standard out will go to the terminal/console. But, if you run grafana via docker, docker will be the parent process and standard out will go to docker. Docker saves this information and you can access it through `docker logs`.

I guess we should change it to console to stay consistent with the config. Standard output is more correct but someone that understands what standard output means should be able to infer what we mean by console too.

Comment (oddlittlebird) [reply docs/sources/installation/configure-docker.md:58]: Agreed. Let's call it "console" so we keep our terminology consistent.

**Comment by aknuds1** [comment]
I guess this is ready to merge @xlson? To pass the Drone CI check you need to merge with master though.

File: docs/sources/installation/configure-docker.md (hunk 1 above)
Comment (aknuds1) [reply docs/sources/installation/configure-docker.md:58]: @oddlittlebird You should change "theDocker" into "the Docker".

**Review by aknuds1 — APPROVED** [review]

**Comment by xlson** [comment]
@aknuds1 yeah, I totally forgot about this one :)

**Pull request merged**